index.js

```diff
@@ -235,19 +253,19 @@ async function bundleInstall(gemfile, lockFile, platform, engine, version) {
   // config
   const path = 'vendor/bundle'
 
-  await exec.exec('bundle', ['config', '--local', 'path', path])
+  await exec.exec('bundle', [`_${bundlerVersion}_`, 'config', '--local', 'path', path])
```

Comment:
It's unfortunate that the `_version_` syntax doesn't support `_~> 1_` or so.

OTOH, adding these extra `_version_` in the action output doesn't seem nice, I'd like to keep commands as simple and close to what a user would do as possible. So if we took this approach, I'd want to only add `_version_` when needed (on Ruby 2+ if bundlerVersion == '1').

But, there seems to be a much simpler way:
```
$ BUNDLER_VERSION=1 bundle version
Bundler version 1.17.3 (2018-12-27 commit d7089abb6)
$ BUNDLER_VERSION=2 bundle version
Bundler version 2.2.0.rc.2 (2020-10-07 commit d7c0467653)
```
That relies on `Gem::BundlerVersionFinder`, but that's probably good enough.

Another way would be to hardcode Bundler 1 to be 1.17.3, but it seems less nice than just setting `BUNDLER_VERSION` in the env for the commands that need it.

Reply:
Oh, didn't knew about `BUNDLER_VERSION=`. I'll check it.